exporter/awss3exporter/README.md

```diff
@@ -32,6 +32,7 @@ The following exporter configuration parameters are supported.
 | `encoding_file_extension` | file format extension suffix when using the `encoding` configuration option. May be left empty for no suffix to be appended.               |             |
 | `endpoint`            | (REST API endpoint) overrides the endpoint used by the exporter instead of constructing it from `region` and `s3_bucket`                                       |             |
 | `s3_force_path_style` | [set this to `true` to force the request to use path-style addressing](http://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html) | false       |
+| `storage_class`         | S3 storageclass | STANDARD
```

Comment:
What are the allowed values for this config key? Is it a known enumeration? Is there a link to docs? Any validation we can perform?

Reply:
The issue (https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35173) mentions STANDARD_IA, is STANDARD valid?

Reply:
Added documentation about the storage class valid APIs and add the all possible options